cmds/replay.c

```diff
@@ -591,13 +603,19 @@ void get_argspec_string(struct uftrace_task_reader *task,
 				print_args("\\\"");
 				while (*p) {
 					char c = *p++;
-					print_escaped_char(c);
+					print_json_escaped_char(c);
 				}
 				print_args("\\\"");
 			}
 			else {
 				print_args("%s", color_string);
-				print_args("\"%.*s\"", slen + newline, str);
+				print_args("\"");
+				char *p = str;
+				while (*p) {
+					char c = *p++;
+					print_escaped_char(c);
+				}
+				print_args("\"");
```

Comment:
Hmm.. This needs to think about multi-byte strings like UTF-8.  I also worry about the performance overhead of calling `snprintf()` for every character.  So I'd suggest adding a check before doing this: it searches the given string to have a character whose value is larger than 128 (0x80).  In this case it thinks the string is in UTF-8 and prints as is.  Otherwise it could try to escape some unprintable characters.

Reply:
Okay. I will check that given string have character whose value is larger than 128 before print it.